src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './TodoList.css';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export class TodoList extends React.Component {
+  render() {
+    const { todos } = this.props;
+
+    return (
+      <ul className="list">
+        {todos.map(({ name, title, completed, id }) => (
+          <li key={id} className="list__item">
+            <>
+              <p className="list__text">{name}</p>
+              <h2 className="list__title">{title}</h2>
```

Comment:
You can create a separate component `Todo` for todo itself

Reply:
The name of the error speaks by itself `react/prefer-stateless-function` means that you should use the function without the state ;)
About components inside `li` - it's not necessary. But it may be useful to create re-usable components for convenience